docs/public/_redirects

```diff
@@ -10,7 +10,7 @@
 https://next.material-ui.com/* https://mui.com/:splat 301!
 
 # Support multiple domains
-https://material-ui.dev/* https://mui.com/:splat 301!
```

Comment:
This also wouldn't have worked without `material-ui.dev` being setup as an alias domain on netlify: 

> For Netlify to redirect from a domain that is not the site’s main custom domain, the domain must be assigned to the site — probably as a domain alias or a branch subdomain.

-- https://docs.netlify.com/routing/redirects/redirect-options/#domain-level-redirects

Reply:
It used to be, I removed it about two months ago.